TODO.md

```diff
@@ -0,0 +1,26 @@
+## TODO
+
+* Add datapoints in provisioning logic for Influx
+* Be more specific in JSON Schema
+* Load things from NextToken when retreiving spot price history
+* Overwrite UserData with temporary taskcluster credentials
+  * Include the API endpoint for the slave to hit the hello
+    endpoint
+  * include the instanceId scope!
```

Comment:
Isn't it the `assume:worker-id:<workerGroup>:<workerId>` scope, of which `capacity` number must be issued...
(This need to be coordinated with docker-worker -- let's discuss setting up a meeting).